db/schema.rb

```diff
@@ -353,15 +352,14 @@
     t.boolean "active_alert", default: false
     t.text "national_insurance_number"
     t.boolean "trn_auto_verified", default: false
-    t.string "feature_flag_id"
     t.string "provider"
     t.string "uid"
     t.jsonb "raw_tra_provider_data"
+    t.string "feature_flag_id"
     t.boolean "get_an_identity_id_synced_to_ecf", default: false
     t.datetime "updated_from_tra_at", precision: nil
     t.string "trn_lookup_status"
     t.boolean "notify_user_for_future_reg", default: false
-    t.boolean "allow_closed_registration"
```

Comment:
We should manually drop this from the databases as it doesn't seem to be used/never had a migration in the first place

Reply:
@slawosz this seems recent, just confirming if this was a wrong commit at some point for closed registration?

Reply:
It could be wrong commit, yes.